src/ORM/Query.php

```diff
@@ -455,13 +429,8 @@ public function contain($associations = null, $override = false)
             $this->clearContain();
         }
 
-        if ($associations === null) {
-            deprecationWarning(
-                'Using Query::contain() as getter is deprecated. ' .
-                'Use getContain() instead.'
-            );
-
-            return $loader->getContain();
+        if ($associations === null && $override === false) {
+            throw new InvalidArgumentException('$associations can be null only $override is true.');
```

Comment:
but `associations === null ` deprecated 
and we have `clearContain()` method for clearing contain 
`->contain( null, true)` is pointless IMO

Reply:
I guess we can remove this check all together and one can just use `contain([], true)` to clear/reset.